multinode-demo/setup.sh

```diff
@@ -23,7 +23,8 @@ default_arg --ledger "$SOLANA_RSYNC_CONFIG_DIR"/ledger
 default_arg --mint "$SOLANA_CONFIG_DIR"/mint-keypair.json
 default_arg --lamports 100000000000000
 default_arg --bootstrap-leader-lamports 424242
-default_arg --lamports-per-signature 1
+default_arg --target-lamports-per-signature 42
```

Comment:
I changed these defaults to ensure that we'd have variable fees for all our testnets.   This is probably going to break some of solana-web3.js and downstream examples, but I'll patch those up after this lands